docs/overview/excel.md

```diff
@@ -68,7 +68,19 @@ More details about sharing and unsharing scripts can be in the article [Sharing
 > [!NOTE]
 > Learn more about how scripts are stored in your OneDrive in [Office Scripts file storage and ownership](script-storage.md).
 
-## Connecting Office Scripts to Power Automate
+### Create script buttons (preview)
+
+[!INCLUDE [Public preview](../includes/preview-note.md)]
+
+Help your colleagues find and run your scripts by adding script buttons to a workbook. With any script, go to the **More options (…)** menu in the script's Details page and select **Add button**. This creates a button in the workbook that runs the associated script when selected. It also shares the script with the workbook, so everyone with write permissions to the workbook can use your helpful automation.
```

Comment:
Minor detail - they could go to the More options menu in either the details or the code editor page of the script.

Should we mention something about how to stop sharing a button or should that be part of a separate set of docs? (To stop sharing, users could either delete the button or select "Stop sharing" in the More options menu.)

Reply:
I'll add those points in. Thanks for raising them.